ci-operator/config/rh-ecosystem-edge/gitstream/rh-ecosystem-edge-gitstream-main.yaml

```diff
@@ -0,0 +1,76 @@
+base_images:
+  go-builder:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.21-openshift-4.16
+binary_build_commands: go build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+images:
+- dockerfile_path: Dockerfile
+  to: gitstream
+promotion:
+  to:
+  - namespace: edge-infrastructure
+    tag: latest
+    tag_by_commit: true
+releases:
+  latest:
+    release:
+      channel: fast
+      version: "4.15"
```

Comment:
Not sure if it's in any way important, but should we use 4.16 instead of 4.15 here?

Reply:
KMM master is currently being tested with 4.15 (because we didn't have any release following 4.16).
The next KMM release 2.2 will target 4.17